.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: ['8', '10']
```

Comment:
It's not really important for this project to support multiple versions concurrently, so the "matrix" abstraction may be more trouble than it's worth. When we do upgrade, we can hop over version 10.

Reply:
Node 12 actually doesn’t work, this started out as as a matrix of 3 versions, and I trimmed it to what actually works.